app/controllers/users_controller.rb

```diff
@@ -8,7 +8,14 @@ def show
   private
 
   def find_user
-    User.find_by(uuid: params[:uuid])
+    # TODO: Not sure how to make this one query
```

Comment:
Two ways:
```
User.joins(:profile).where("profiles.city = ? OR users.uuid = ?", params[:slug], params[:slug]) =>
SELECT  "users".* FROM "users" INNER JOIN "profiles" ON "profiles"."user_id" = "users"."id" WHERE (profiles.username = '766b645b-3390-49af-b1b1-68337f90ee4d' OR users.uuid = '766b645b-3390-49af-b1b1-68337f90ee4d') LIMIT $1  [["LIMIT", 11]]
```
or
```
User.joins(:profile).where(profiles: {city: params[:slug]}).or(User.joins(:profile).where(uuid: params[:slug])) =>
SELECT  "users".* FROM "users" INNER JOIN "profiles" ON "profiles"."user_id" = "users"."id" WHERE "profiles"."city" = $1 AND "users"."uuid" = $2 LIMIT $3  [["city", "766b645b-3390-49af-b1b1-68337f90ee4d"], ["uuid", "766b645b-3390-49af-b1b1-68337f90ee4d"], ["LIMIT", 11]]
```

Stinks that you need the `joins` in the second query so they're compatible. The queries are identical except for how they're substituting values in for the filters.